### DISCUSSION TITLE
live server --> opens tab: listing directory /

### DISCUSSION THREAD

**Discussion by Alexander-Iantbelidze** [neuefische/questions#764] [Web: Self-directed learning] [answered]
hi, i cannot run the live server on github (vs code). 

it pop's up like this: 
![Bildschirmfoto 2023-10-19 um 15 29 56](https://github.com/neuefische/questions/assets/147614805/31128c17-faaf-4ec4-b075-bd3482b79a1f)

and i just see a search input field.

What i have already tried?

1. checked the correct link to the css file:
![Bildschirmfoto 2023-10-19 um 15 32 10](https://github.com/neuefische/questions/assets/147614805/624a36ab-6da9-41a2-94bd-8537b96c54b0)

2. Checked folders:

![Bildschirmfoto 2023-10-19 um 15 34 35](https://github.com/neuefische/questions/assets/147614805/5708000f-dea5-4a32-8834-1cad2d523ad8)

3. Reinstalled live server extension

Still not working...

Can anyone help me?

Thanks!

**Comment by mbosselmann** [comment, marked as answer]
Hi Alexander,
I think the problem is where your files are located as all of them are inside of the `.vscode` folder. I recommend to move the `index.html` file to the root (same level as the `.vscode` folder) and remove the then empty HTML folder. And you should also move the `CSS` folder one level up.

I hope this does help! :)

**Reply by Alexander-Iantbelidze** [reply]
Hi Mareike,

.vscode/index.html
.vscode/CSS/styles.css

is that correct?

**Reply by moonwave99** [reply]
Hey Alex,

try like this:
```
.vscode
  settings.json
CSS
  styles.css
HTML
  index.html
```

**Reply by Alexander-Iantbelidze** [reply]
Hey,

i tried, but still not working..

.vscode/settings.json
.vscode/css/styles.css
.vscode/html/index.html

Thanks!
